packages/mobile/src/components/share-drawer/utils.ts

```diff
@@ -35,8 +36,17 @@ export const getContentUrl = (content: ShareContent) => {
   }
 }
 
-const getShareHandle = async (handle: string) => {
-  const { twitterHandle } = await audiusBackendInstance.getSocialHandles(handle)
+const getTwitterShareHandle = async ({
```

Comment:
I'm a little lost on this new pattern of passing in `twitterHandle` separately?

Reply:
it seems like the change was made so that we use whatever the user was verified with, but that should be coming from the backend probably so I think this unnecessary?

Reply:
handled.